File: WebApi/ClientApp/src/app/models/anamnese.ts
export class Anamnese
{
    id?: number;
    name: string;
    date: Date | string;
    countOfPositivAnswers?: number;
    chapters: AnamneseChapter[];
    patientId: number | null;
}

export interface AnamneseQuestion
{
    id: number | null;
    type: string;
    value: string;
    label: string;
    metaInfo: string;
    textPrefix: string;
    textValue: string;
    chapterId: number | null;
}


export interface AnamneseChapter
{
    id: number | null;
    name: string;
    questions: AnamneseQuestion[];
    anamneseId: number | null;

}

File: rl-contract/Models/Documents/Anamnese.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace rl_contract.Models
{
    public class Anamnese
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int? Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public int? CountOfPositivAnswers { get; set; }

        public List<AnamneseChapter> Chapters { get; set; }

        public int? PatientId { get; set; }
        /*public Patient Patient { get; set; }*/

    }
}
